src/App.js

```diff
@@ -1,25 +1,30 @@
-import logo from './logo.svg';
+import React from 'react';
```

Comment:
correct me if I'm wrong, but I believe the newer version of React doesn't necessarily need you to import `React` anywhere else than the `index.js`